orangecontrib/imageanalytics/image_embedder.py

```diff
@@ -193,11 +196,21 @@ def _send_to_server(self, file_paths, image_processed_callback):
         )
 
     def _load_image_or_none(self, file_path):
-        try:
-            image = open_image(file_path)
-        except IOError:
-            log.warning("Image skipped (invalid file path)", exc_info=True)
-            return None
+        parsed_url = urlparse(file_path)
+        if parsed_url.scheme == 'http' or parsed_url.scheme == 'https':
+            response = requests.get(file_path, stream=True)
```

Comment:
Of course.

Since all exceptions are unexpected in this case, handling them in a similar, broad manner is ok:
```py
try:
    response = requests.get(file_path, stream=True)
except requests.exceptions.RequestException as e:  # this is the base requests exception
    self.Error.request_failed(' '.join(e.args))  # See the link below
    log.error(...)
    return
```

See about raising errors and warning flash bars in widgets: https://orange-development.readthedocs.io/tutorial-utilities.html#multiple-messages